openrtb_ext/imp_stroeercore.go

```diff
@@ -0,0 +1,5 @@
+package openrtb_ext
+
+type ExtImpStroeercore struct {
```

Comment:
You may use the preferred casing of your adapter for this type, ie `type ExtImpStroeerCore struct {`